docs/blog/newsletter-october-2017.rst

```diff
@@ -0,0 +1,81 @@
+
+*************************
+The true meaning of "API"
+*************************
+What exactly are we referring to when we say "API"? While the initialism itself stands for Application Programming Interface, it's a more complicated question than it seems. When it came up this month, it sparked a lively conversation that suggested that API might mean different things to different people in different contexts.
+
+Most of the conversation revolved around that idea that, when we say "API" we assume that our reader (or listener) will inherently understand the type of API we're talking about. But that might not be the case as often as you think. If you work with REST APIs, it's likely you'll assume that API has a silent "REST" in front of it. But if you've spent your career working with library APIs, that will probably be the meaning you default to.
+
+There was some question of whether any of this mattered. Whether it's a library, REST, SOAP, or other type of API, they're all still "methods of communication between various software components" (thank you `wikipedia <https://en.wikipedia.org/wiki/Application_programming_interface>`_). But as evidenced by the length and depth of the conversation, it's definitely a topic worth clarifying. Handily enough, one of our Prague speakers, Garen Torikian discusses has a slide on this very topic in his presentation, "Finally! Trustworthy and Sensible API Documentation with GraphQL." Stay tuned for the video recording!
+
+In the meantime, check out #documenting-apis or #apithedocs for more conversations about APIs!
+
+******************************************
+Proofreading and copyediting your own work
+******************************************
+If you're a lone writer, you've probably been the unenviable position of not having an editor around to proofread, copyedit, or otherwise sanity-check your writing. It's a perennial dilemma, and one that surfaces regularly on Slack. This month produced a particularly good batch of suggestions and resources. To help you clean up your copy when your only editor is you, here's some tips and tricks from the Write the Docs hivemind:
+
+* Put your writing away and review it at least a day later.
+* Print it out on paper and read/edit in the new format.
+* Read each sentence independently. Try reversing the order of sentences, starting with the last one and working your way backward to the first. Or reverse the order of paragraphs.
+* Listen to it being read aloud, either by you or by a device. Text-to-Speech tools are widely available.
```

Comment:
Maybe replace "it" with "your writing" here?

Reply:
👍